wrapper/src/gnatcuda_wrapper.adb

```diff
@@ -227,25 +259,55 @@ begin
             LLVM_Args (LLVM_Arg_Number) := new String'(Arg);
          elsif Arg = "-v" then
             Put_Line ("Target: cuda");
-            --  ??? temporary hard coded version numbers
             Put_Line ("cuda-gcc version "
               & gnatvsn.Library_Version
               & " (for GNAT Pro "
               & gnatvsn.Gnat_Static_Version_String
               & ")");
+            Put_Line ("CUDA Installation: " & CUDA_Root);
 
             Verbose := True;
             LLVM_Arg_Number := @ + 1;
             LLVM_Args (LLVM_Arg_Number) := new String'(Argument (J));
          elsif Arg'Length > 9 and then Arg (1 .. 9) = "-mcpu=sm_" then
             GPU_Name := new String'(Arg (10 .. Arg'Last));
+         elsif Arg'Length > 9 and then Arg (1 .. 17) = "-mcuda-libdevice=" then
```

Comment:
Length > 17

Maybe is it a sign that this check needs to be refactored by adding some `function Get_Suffix (S: String; Prefix: String; Suffix : out String_Access) return Boolean`?

Reply:
Good catch - and good idea. I'll refactor.